.github/workflows/Dockerfile.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
   push:
     # Push image to registry after build for pull requests from a local branch.
-    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository }}
+    if: ${{ github.ref == 'refs/heads/master' }} || ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
Prefer `github.ref_protected` over checking master exactly. I think we'd generally want to run pushes on other protected branches too if they develop. So I try to use it instead of hardcoding branch names.

Reply:
nice optimization, done, thanks.